### DISCUSSION TITLE
Where is Sharpgen 2.1.2 beta?

### DISCUSSION THREAD

**Discussion by Steph55** [amerkoleci/Vortice.Windows#411] [General]
Your software is well done and very useful, thank you very much!

However, I cannot use the last builds because Sharpgen.Runtime 2.1.2-beta and Sharpgen.Runtime.COM 2.1.2-beta are both required and not available on Nuget. Where can we find them?

Also, here is one minor typographical error to correct if not already corrected, in the ConvolveMatrix D2D effect. The property KernelUnitLehgth should be renamed KernelUnitLength.

**Comment by amerkoleci** [comment]
The SharpGenTools packages are on nuget: https://www.nuget.org/packages/SharpGen.Runtime/2.1.2-beta 

Make sure to enable prelease packages in Visual Studio

**Reply by Steph55** [reply]
Oh! Thanks a lot!
